cmd/tendermint/commands/reset_priv_validator.go

```diff
@@ -31,6 +32,22 @@ func MakeResetAllCommand(conf *config.Config, logger log.Logger) *cobra.Command
 	return cmd
 }
 
+// MakeResetAllCommand constructs a command that removes the database of
+// the specified Tendermint core instance.
+func MakeResetStateCommand(conf *config.Config, logger log.Logger) *cobra.Command {
+	var keyType string
+
+	cmd := &cobra.Command{
+		Use:   "unsafe-reset-state",
+		Short: "(unsafe) Remove all the data and WAL",
```

Comment:
this is probably wrong

Reply:
do we want to also remove wal?

Reply:
oh I was just looking at safe/unsafe langauge. 

I the WAL isn't going to help anything to keep it around